src/main/java/com/google/codeu/data/User.java

```diff
@@ -16,4 +28,12 @@ public String getEmail() {
   public String getAboutMe() {
     return aboutMe;
   }
+
+  public ArrayList getMentions() {
+    return mentions;
+  }
+
+  public void addMention(String messageId) {
+    if (mentions.contains(messageId) == false) mentions.add(messageId);
```

Comment:
This (and not really having a purpose for iterating through messages) makes me think that a HashSet might be the more appropriate data structure?

Reply:
Yeah...when your application scales this will become a bottleneck, however, fine to keep like this for now.